packages/ui/lib/components/Display/Address.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { Meta, StoryFn } from '@storybook/react'
+import { Address, AddressProps } from './Address'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { Toaster } from 'react-hot-toast'
+
+const queryClient = new QueryClient()
+
+const withQueryClient = (Story: StoryFn) => (
+  <QueryClientProvider client={queryClient}>
+    <Toaster position="top-center" />
+    <Story />
+  </QueryClientProvider>
+)
+
+export default {
+  title: 'Address',
+  component: Address,
+  argTypes: {
+    address: { control: 'text' },
+    showExternalLink: { control: 'boolean' },
+    showCopyIcon: { control: 'boolean' },
+    showENSName: { control: 'boolean' },
```

Comment:
I think we need to make this a bit more flexible to allow more than just ENS resolution (like support for Basenames, cb.id and eventually other names like Unlimited Domains... etc)